plugins/view-cert.yaml

```
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: view-cert
+spec:
+  version: v0.0.2
+  homepage: https://github.com/lmolas/kubectl-view-cert
+  shortDescription: View certificate information stored in secrets
+  description: |
+    kubectl krew plugin to view certificate information stored in secrets.
+    Supported secret types are kubernetes.io/tls. But you can parse any type of secret by specifying secret name and secret key to read.
```

- No need to mention this is a kubectl plugin :wink:
- Please wrap at ~80 chars so that it prints nicely on wide and narrow screens.
- You might want to add what information from the certs you display (expiration date, issuer,..)

```suggestion
    View certificate information stored in secrets.
    Supported secret types are kubernetes.io/tls. But you can parse any
    type of secret by specifying secret name and secret key to read.
```